### ISSUE TITLE
Tensorflow dataloader adding channel dim

### ISSUE THREAD

**Issue by swpenninga** [tue-bmd/zea#252]
The code linked below always maps a channel dim if your data isnt exactly (B, H, W). 
I guess this assumes we only work with B-mode. I personally think it should be changed to an argument.

https://github.com/tue-bmd/zea/blob/64386323696f27dd939bea62cfa10154960996c1/zea/backend/tensorflow/dataloader.py#L307-L310
